content/en/docs/ops/diagnostic-tools/multicluster/index.md

```diff
@@ -49,6 +57,9 @@ the steps for every cluster.
 
 ### Step-by-step Diagnosis
 
+If none of the above solved the issue breaking cross-cluster load-balancing, this will require more in-depth
```

Comment:
Maybe something like "If you've gone through the sections above and are still having issues with cross-cluster load-balancing, then it's time to dig a little deeper." ... wdyt?

Reply:
Yeah, that sounds better.